### ISSUE TITLE
unbindService() does not close OkHttp responde body

### ISSUE THREAD

**Issue by HugoGresse** [lingochamp/FileDownloader#68]
When calling `FileDownloader.getImpl().unBindService();`, the OkHttp response body is not closed, following log is displayed:

```
W/okhttp3.OkHttpClient: A connection to https://google.com/ was leaked. Did you forget to close a response body?
```

**Comment by Jacksgong** [comment]
Thank you so much, I will fix it immediately!

**Comment by Jacksgong** [comment]
Fixed in [v0.2.1](https://github.com/lingochamp/FileDownloader/releases/tag/v0.2.1).
